core/src/main/java/io/smallrye/asyncapi/core/runtime/io/schema/SchemaFactory.java

```diff
@@ -255,19 +254,19 @@ public static Schema readSchema(final AnnotationScannerContext context, Schema s
         schema.setExtensions(ExtensionReader.readExtensions(context, annotation));
 
         schema.setProperties(SchemaFactory.<AnnotationInstance[], Map<String, Schema>> readAttr(annotation,
-            SchemaConstant.PROP_PROPERTIES, properties -> {
-                if (properties == null || properties.length == 0) {
-                    return null;
-                }
-                Map<String, Schema> propertySchemas = new LinkedHashMap<>(properties.length);
-                for (AnnotationInstance propAnnotation : properties) {
-                    String key = JandexUtil.value(propAnnotation, SchemaConstant.PROP_NAME);
-                    Schema value = readSchema(context, new SchemaImpl(), propAnnotation, Collections.emptyMap());
-                    propertySchemas.put(key, value);
-                }
-
-                return propertySchemas;
-            }, defaults));
+                SchemaConstant.PROP_PROPERTIES, properties -> {
```

Comment:
please check indentation is wanted or unwanted

Reply:
it is